gatorgrade/main.py

```diff
@@ -0,0 +1,21 @@
+"""Use Typer to run gatorgrade to run the checks and generate the yml file."""
+import typer
+from pathlib import Path
+
+app = typer.Typer(add_completion=False)
+FILE_DIR = "config"
+FILE = "gatorgrade.yml"
+
+
+@app.callback(invoke_without_command=True)
+def gatorgrade(file_dir: Path = typer.Option(f"{FILE_DIR}/{FILE}")):
```

Comment:
can you please make sure that the flag of this argument is specified in the Typer option. Thanks!